jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/assignment/AssignmentService.java

```diff
@@ -20,7 +20,8 @@
 import org.kie.api.task.model.Task;
 
 public interface AssignmentService {
-    
+    void setEnabled(boolean enabled);
```

Comment:
is this really required? so far, enabling AssigmentService is done through the system property "org.jbpm.task.assignment.enabled"

https://github.com/kiegroup/jbpm/blob/12ac6e7b3c68939b739c06152112d6c034295672/jbpm-human-task/jbpm-human-task-core/src/main/java/org/jbpm/services/task/assignment/impl/AssignmentServiceImpl.java#L42

By adding this new method to the interface, we're enabling a different way of getting this done, so not really sure if this is needed?

Reply:
it is convenient for testing purposes.